src/screens/simulation/Chart.js

```diff
@@ -19,137 +22,218 @@ const useStyles = makeStyles({
-                    fontSize: 18,
+        axis: {
+            y: {
+                type: yType,
+                tick: {
+                    format: (value) => Math.round(value),
                 },
             },
-        ],
-        xAxes: [
-            {
-                scaleLabel: {
-                    display: true,
-                    labelString: t('chart.x_scale_label'),
-                    fontSize: 18,
+            x: {
+                type: 'timeseries',
+                tick: {
+                    count: 8,
+                    format: '%d/%m/%Y',
                 },
             },
-        ],
-    },
-});
-
-const Chart = ({ values, startDate }) => {
-    const classes = useStyles();
-    const t = useTranslate();
-    const theme = useTheme();
-    const medium = useMediaQuery(theme.breakpoints.up('md'));
-    const xmedium = useMediaQuery('@media (min-width:1400px)');
-    const large = useMediaQuery(theme.breakpoints.up('lg'));
-    const midlarge = useMediaQuery('@media (min-width:1400px)');
-    const xlarge = useMediaQuery(theme.breakpoints.up('xl'));
-
-    let chartSize;
-    if (xlarge) chartSize = 725;
-    else if (midlarge) chartSize = 650;
-    else if (large) chartSize = 575;
-    else if (xmedium) chartSize = 475;
-    else if (medium) chartSize = 425;
-    else chartSize = 325;
-
-    const lineData = data({ t, values, startDate });
+        },
+        grid: {
+            x: {
+                lines: timeframes.map((timeframe) => ({
+                    value: format(timeframe.date, 'yyyy-MM-dd'),
+                    text: timeframe.label,
+                })),
+            },
+        },
+        legend: {
+            inset: {
+                anchor: 'top-right',
+                x: 20,
+                y: 10,
+                step: 2,
+            },
+        },
+        zoom: {
+            enabled: true,
+        },
+        size: { height: chartSize * CHART_HEIGHT_RATIO, width: chartSize },
+        tooltip: {
+            format: {
+                title: (date) => format(date, 'dd/MM/yyyy'),
+                value: (value) => Math.round((value + Number.EPSILON) * 100) / 100,
+            },
+        },
+    };
 
     return (
         <div className={classes.root}>
-            <div className={classes.container} style={{ width: chartSize, height: chartSize }}>
-                <Line
-                    data={lineData}
-                    width={chartSize}
-                    height={chartSize}
-                    options={getOptions(t)}
-                />
+            <div className={classes.container} style={config.size}>
+                <C3Graph config={config} />
+                <Button
+                    variant="contained"
+                    className={classes.yAxisToggleButton}
+                    onClick={handleYTypeToggle}
+                >
+                    Affichage {yType === 'linear' ? 'Linéaire' : 'Logarithmique'}
```

Comment:
the button is not clear for me. Does it mean "Go to linear scale" ou "I am in linear sclae please switch"
May be add 'linear scale' or 'log scale', close to the y axis